### PR TITLE
OOB invitations

### PR DESCRIPTION
**Pull request by loneil**
Allow choice between OOB connection type and connections/1.0 for creating an invitation.
Supports simple invitation protocol, can consider expanding to more complex handshake procotol stuff in the future but that would be best with a URL shortening plan (QR code definitely could have issues)

### CODE REVIEW

**Comment by jamshale** [comment]
The failing tests were probably a bit to fragile. Changing the table columns would make them fail.

**Comment by loneil** [comment]
> The failing tests were probably a bit to fragile. Changing the table columns would make them fail.
> 
> Could just remove the failing test or update it with the expected column value.

@jamshale yeah sorry had that fixed but got sidetracked with other PRs and hadn't requests a review on this one yet.
Should be good to go to take a look now, thanks.

File: services/tenant-ui/frontend/src/components/connections/createConnection/CreateConnectionForm.vue [hunk 1]
```diff
@@ -1,7 +1,59 @@
...
+            v-model="isOob"
+            input-id="type1"
+            name="type"
+            :value="false"
+          />
+          <label for="type1" class="ml-2">{{
+            $t('connect.invitation.typeConnections')
+          }}</label>
+        </div>
+        <div>
```
Comment (jamshale) [inline services/tenant-ui/frontend/src/components/connections/createConnection/CreateConnectionForm.vue:18]: This extra div needed? The other group doesn't have one. 
It's only a tiny bit of code but these 2 elements consisting of title+radio+label+radio+radio look like they could be the same component.

File: services/tenant-ui/frontend/test/components/connections/Invitations.test.ts [hunk 2]
```diff
@@ -20,16 +20,17 @@ describe('Invitations', () => {
...
 
   test('table body is rendered with expected values', async () => {
     const wrapper = mountInvitations();
-    const expectedTexts = ['', 'test.alias', 'once'];
+    const expectedTexts = ['', 'test.alias', 'once', 'connections/1.0'];
 
     // td is an expected text or valid date
     wrapper.findAll('tbody td').forEach((td) => {
       const text = td.text();
+      console.log(text);
```
Comment (jamshale) [inline services/tenant-ui/frontend/test/components/connections/Invitations.test.ts:33]: Probably left over from checking the test html.

**Review by jamshale — APPROVED** [review]

File: services/tenant-ui/frontend/src/components/connections/createConnection/CreateConnectionForm.vue (hunk 1 above)
Comment (loneil) [reply services/tenant-ui/frontend/src/components/connections/createConnection/CreateConnectionForm.vue:18]: The divs around the Radio + Label elements? Without those things end up in-line
image

I might be misunderstanding

File: services/tenant-ui/frontend/test/components/connections/Invitations.test.ts (hunk 2 above)
Comment (loneil) [reply services/tenant-ui/frontend/test/components/connections/Invitations.test.ts:33]: Thx! Fixed

File: services/tenant-ui/frontend/src/components/connections/createConnection/CreateConnectionForm.vue (hunk 1 above)
Comment (jamshale) [reply services/tenant-ui/frontend/src/components/connections/createConnection/CreateConnectionForm.vue:18]: Oh, I see. Just the top one needed a margin.

**Review by jamshale — APPROVED** [review]

**Pull request merged**